cmd/sonobuoy/app/args.go

```diff
@@ -117,6 +117,9 @@ func AddKubeConformanceImageVersion(imageVersion *image.ConformanceImageVersion,
 
 	*imageVersion = image.ConformanceImageVersionAuto
 	flags.Var(imageVersion, "kube-conformance-image-version", help)
+	if err := flags.MarkDeprecated("kube-conformance-image-version", "Use --kubernetes-version instead."); err != nil {
+		panic("Failed to setup flags properly")
```

Comment:
This would happen every time you run so I dont think its a risk at all to panic here.